index.html

```diff
@@ -1201,15 +1201,15 @@ <h2>Core Vocabulary Definitions</h2>
           "td-context-ns-thing-mandatory">The <code>@context</code>
           name-value pair <em class="rfc2119" title=
           "MUST">MUST</em> contain the anyURI
-          <code>https://www.w3.org/2022/wot/td/v1.1</code> either
+          <code>https://www.w3.org/2019/wot/td/v1</code> either
           directly when of type <code>anyURI</code> or as first
           element when of type <a href="#dfn-array" class=
           "internalDFN" data-link-type="dfn">Array</a>.</span>
           <span class="rfc2119-assertion" id=
           "td-context-ns-thing-optional">When <code>@context</code>
           is an <a href="#dfn-array" class="internalDFN"
           data-link-type="dfn">Array</a>, the anyURI
-          <code>https://www.w3.org/2022/wot/td/v1.1</code> <em class=
+          <code>https://www.w3.org/2019/wot/td/v1</code> <em class=
```

Comment:
Are these changes intended? cause in https://github.com/w3c/wot-thing-description/pull/1333 there are the same changes but from 2019 to 2022 🤔

Reply:
many thanks, I fixed it.